pkg/virt-config/configuration.go

```diff
@@ -371,7 +380,37 @@ func (c *ClusterConfig) HasDataVolumeAPI() bool {
 	objects := c.crdInformer.GetStore().List()
 	for _, obj := range objects {
 		if crd, ok := obj.(*extv1.CustomResourceDefinition); ok && crd.DeletionTimestamp == nil {
-			if isDataVolumeCrd(crd) {
+			if isServiceMonitor(crd) {
```

Comment:
This change does not look right for me. `HasDataVolumeAPI` should check for DataVolume CRD with `isDataVolumeCrd`